gst/nnstreamer/tensor_query/nnstreamer_edge_common.c

```diff
@@ -12,6 +12,118 @@
 
 #include "nnstreamer_edge_common.h"
 
+/**
+ * @brief Create nnstreamer edge event.
+ */
+int
+nns_edge_event_create (nns_edge_event_e event, nns_edge_event_h * event_h)
+{
+  nns_edge_event_s *ee;
+
+  if (!event_h) {
+    nns_edge_loge ("Invalid param, event_h should not be null.");
+    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+  }
+
+  if (event <= NNS_EDGE_EVENT_UNKNOWN) {
+    nns_edge_loge ("Invalid param, given event type is invalid.");
+    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+  }
+
+  ee = g_try_new (nns_edge_event_s, 1);
+  if (!ee) {
+    nns_edge_loge ("Failed to allocate memory for edge event.");
+    return NNS_EDGE_ERROR_OUT_OF_MEMORY;
+  }
+
+  memset (ee, 0, sizeof (nns_edge_event_s));
```

Comment:
use g_try_new0 and remove line 39? :)

Reply:
I changed this, g_try_new() > malloc() as glib would be removed later. :)